packages/mobile/src/alert/actions.ts

```diff
@@ -65,12 +65,9 @@ export const showErrorInline = (error: ErrorMessages, i18nOptions?: TOptions): S
 export function showErrorOrFallback(error: any, fallback: ErrorMessages) {
   if (error && Object.values(ErrorMessages).includes(error.message)) {
     return showError(error.message)
-  } else {
-    CeloAnalytics.track(CustomEventNames.error_fallback, {
```

Comment:
Any reason to remove this? Error_fallback was helping to catch missing errors- for example if verification failed we could see a more detailed error message than the one showed to users

Reply:
Oh my bad, I missed this use case and will add it back. Was just trying to cut down on repetition because we are already logging errors with context specific error events, on error boundary, on showError, and also here.

Thinking about if there's a better way to do this for the next PR